package.json

```diff
@@ -26,7 +26,7 @@
   "//": "in backend directory : Install dependencies",
 	"//": "The message 'Entering directory' is common to e.g. make -w, and recognised by emacs compilation-mode to set default-directory",
   "scripts": {
-    "postinstall" : "ln -s ../frontend/dist backend/client && ln -s ../../../../backend/common/utilities/interval-overlap.js frontend/app/utils/draw/.",
+    "postinstall" : "ln -s frontend/dist backend/client && ln -s backend/common/utilities/interval-overlap.js frontend/app/utils/draw/.",
```

Comment:
If these were hard links then removing the .../ would be correct, but these are soft links so the link value is relative to the directory which the link is placed in  (and they are directories so hard links would not be an option).

Reply:
Embarrassing, I'd forgotten symbolic links resolve that way.

Reply:
no worries - you had me wondering if the build was failing due to Windows path differences :-)

I might at some point move more of the build logic into node.js so at least it will be multi-platform (although MSWin is the only platform which differs, and now it has WSL, so it is not a priority.  and shell commands are RAD and flexible)